pinot-controller/src/main/resources/landing/index.html

```diff
@@ -0,0 +1,80 @@
+</head>
+<body>
+
+<nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Pinot Controller</a>
+    </div>
+    <div id="navbar" class="navbar-collapse collapse">
+
+      <!-- <form class="navbar-form navbar-right">
+        <div class="form-group">
+          <input type="text" placeholder="Email" class="form-control">
+        </div>
+        <div class="form-group">
+          <input type="password" placeholder="Password" class="form-control">
+        </div>
+        <button type="submit" class="btn btn-success">Sign in</button>
+      </form> -->
+    </div><!--/.navbar-collapse -->
+  </div>
+</nav>
+
+<!-- Main jumbotron for a primary marketing message or call to action -->
+<div class="jumbotron">
+  <div class="container">
+    <h1>Welcome to Pinot!</h1>
+    <p>You're currently connected to a Pinot controller, which handles the cluster-wide coordination for all Pinot nodes. Most humans tend to find the following features useful:</p>
+    <p><a class="btn btn-primary btn-lg" href="/query" role="button">Open query console &raquo;</a> <a class="btn btn-primary btn-lg" href="/help" role="button">Open REST API &raquo;</a></p>
+  </div>
+</div>
+
+<div class="container">
+  <!-- Example row of columns -->
+  <div class="row">
+    <div class="col-md-4">
+      <h2>Query console</h2>
+      <p>A place to conveniently run queries on the Pinot cluster. This is meant to be used for debugging, production queries should be sent directly to the appropriate brokers, to avoid having the controller forward the query to the broker.</p>
+      <p><a class="btn btn-default" href="/query" role="button">Query console &raquo;</a></p>
+    </div>
+    <div class="col-md-4">
+      <h2>Pinot REST API</h2>
+      <p>An interactive documentation for the Pinot REST API with a swanky Swagger-generated page. Keep in mind that your actions here will directly affect the cluster, so be mindful of other tenants.</p>
+      <p><a class="btn btn-default" href="/help" role="button">REST API &raquo;</a></p>
+    </div>
+<!--    <div class="col-md-4">
+      <h2>Heading</h2>
+      <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
+      <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+    </div> -->
+  </div>
+
+  <hr>
+
```

Comment:
suggestion: add a link to the product page on github.
Project page : Github link
For any issues/comments: contact